Arifa

BasicProgrammer1996 opened the pull request:
added the file manager to the project and added code to read and save files

BasicProgrammer1996 committed:
Saving and loadinig

the in here will to save model data

BasicProgrammer1996 committed:
save

save load info on model

kevin-kruusi commented:
Try to avoid pushing things like the .db files they are generated files and will not be needed on the repo.

kevin-kruusi commented:
the .ipch files are pre compiled header files that only exist when you have the editor open and dont need to push them. Focus only only pushing files you intentionally edited.

kevin-kruusi commented:
None of the files in this PR have anything to do with the FileManager.

kevin-kruusi commented:
Suggest closing this pull request and only adding the files you edit in a new pull request.

BasicProgrammer1996 committed:
added JSON code

Added the json.hhp file to the the project so I can use json imeplentaion in the project so I will be able read  and write in json

BasicProgrammer1996 committed:
Update FileManager.cpp

Updated the FileManager to reader  files

BasicProgrammer1996 committed:
Updating Filemanager

Im currently doing edits to make filemanger work so far im able to the gameobject from the Scence0 file but cant get pos and vel

BasicProgrammer1996 committed:
update file Manager

did some editing to the file manager

BasicProgrammer1996 committed:
add stuff to git nore

added severl files to git nore
